pkgs/development/python-modules/ipyvue/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, pythonOlder
+, buildPythonPackage
+, fetchPypi
+  # Python Inputs
+, ipywidgets
+  # check inputs
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "ipyvue";
+  version = "1.3.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "01kp7d8iyz04d0m4njhv8rg72daqmgd93bdljwyffzxg71pwq92a";
+  };
+
+  propagatedBuildInputs = [
+    ipywidgets
+  ];
+
+  doCheck = false;  # No tests in package or GitHub
```

verify that it's not trivially broken
```suggestion
  doCheck = false;  # No tests in package or GitHub
  pythonImportsCheck = [ "ipyvue" ];
```

---

Thanks, I forgot while doing it quick & dirty. will add.

---

done.